lib/app/utils/assertions.rb

```diff
@@ -260,5 +262,13 @@ def assert_deny_previous_tls(uri)
           )
       end
     end
+
+    def assert_valid_http_uri(uri, message = nil)
+      error_message = message
+      if error_message.nil?
```

Comment:
`error_message = message || "\"#{uri}\" is not a valid URI"`

Reply:
change incorporated and pushed